frontend/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -892,6 +893,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				"EXCLUDED_ARCHS[sdk=*]" = armv7;
```

Comment:
Is this correct?

Reply:
Some lines above/below there is also another change. Perhaps check the whole project.pbxproj for unwanted changes

Reply:
everything fine :)